src/bin/pg_autoctl/pgctl.c

```diff
@@ -252,6 +252,67 @@ config_find_pg_ctl(PostgresSetup *pgSetup)
 }
 
 
+/*
+ * find_extension_control_file ensures that the extension is present in the
+ * given Postgres installation. This does the equivalent of:
+ * ls -l $(pg_config --sharedir)/extension/pg_stat_statements.control
+ */
+bool
+find_extension_control_file(PostgresSetup *pgSetup, char *extName)
+{
+	char pg_config_path[MAXPGPATH] = { 0 };
+	char extension_path[MAXPGPATH] = { 0 };
+	char *share_dir;
+	char extension_control_file_name[MAXPGPATH] = { 0 };
+	char *lines[1];
+	Program prog;
+
+	log_info("Checking if the %s extension is installed", extName);
```

Comment:
I think we can make it a `log_debug`, we don't expect this to either fail or take a long time...

Reply:
bump.